fuzzer/svc_dec_fuzzer.cpp

```diff
@@ -329,7 +329,8 @@ void Codec::decodeHeader(const uint8_t *data, size_t size)
         s_video_decode_op.s_ivd_video_decode_op_t.u4_size = sizeof(s_video_decode_op);
 
         ret = ivd_api_function(mCodec, (void *) &s_video_decode_ip, (void *) &s_video_decode_op);
-        (void(ret));
+        if (ret != IV_SUCCESS)
```

Comment:
please remove this change. fuzzer shouldn't have such error checks.
The idea is to let the fuzzer continue to test the cases where input is queued even after previous errors.

Reply:
done.

Reply:
Reverted this change and ammended the commit.